EpiAware/src/EpiLatentModels/EpiLatentModels.jl

```diff
@@ -3,19 +3,29 @@ Module for defining latent models.
 """
 module EpiLatentModels
 
+using Base: AbstractBroadcasted
```

Comment:
We aren't doing [custom broadcasting methods](https://docs.julialang.org/en/v1/manual/interfaces/#man-interfaces-broadcasting) so I'm not sure about having this as part of the `AbstractBroadcasted` type tree; unless thats a future goal. Its probably harmless though.